activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Define `unfreeze_time` alias method of `travel_back` in `ActiveSupport::Testing::TimeHelpers`.
+
+    The alias is provided for symmetry with `freeze_time`
```

Comment:
This sentence is missing a full stop.

Reply:
Punctuation is important too - now added.